### PR TITLE
docs: tracker side effects [DOCDHIS2-79]

### PR DESCRIPTION
**Pull request by zubaira**

### CODE REVIEW

File: src/commonmark/en/content/developer/web-api/new-tracker.md [hunk 1]
```diff
@@ -1103,10 +1103,16 @@ Additionally, program rules can also result in side-effects, like send and sched
 
 <!--DHIS2-SECTION-ID:webapi_nti_side_effects-->
 
-  * Describe what side-effects are
-  * Note that side-effects can fail even if import succeeds. But not the other way around
-  * Describe what side-effects we have and what they do (Table?)
-  * Any configuration that affects side effects.
+  * Side effects define tasks which can be started in separate thread so that actuall import does not get affected by the outcome of the side-effects. As name suggest these are
```
Comment (vilkg) [inline src/commonmark/en/content/developer/web-api/new-tracker.md]: actuall - actual

File: src/commonmark/en/content/developer/web-api/new-tracker.md [hunk 2]
```diff
@@ -1103,10 +1103,16 @@ Additionally, program rules can also result in side-effects, like send and sched
...
-  * Describe what side-effects we have and what they do (Table?)
-  * Any configuration that affects side effects.
+  * Side effects define tasks which can be started in separate thread so that actuall import does not get affected by the outcome of the side-effects. As name suggest these are 
+  consequences of import but are not directly related to import. Note that side-effects can fail even if import succeeds. But not the other way around.
+  * For now these include
+  
+  |Side Effects|Supported|Description|
+  |---|:---:|---|
+  |**Tracker Notification**|**X**| Tracker updates can trigger notifications. Updates which trigger notifications are **enrollment**, **event update**, **event or enrollment completion**. |
+  |**ProgramRule Notification**|**X**| Notifications can also be the result of certain program rule execution. Note that these notifications are part of rule effects which are created as a result of rule-engine exeuciton in pre-validation phase.|
```
Comment (vilkg) [inline src/commonmark/en/content/developer/web-api/new-tracker.md]: exeuciton - execution

File: src/commonmark/en/content/developer/web-api/new-tracker.md [hunk 3]
```diff
@@ -1103,10 +1103,16 @@ Additionally, program rules can also result in side-effects, like send and sched
...
+  
+  * There are certain configurations which can control execution of side effects. `skipSideEffects` flag can be set in `TrackerImportParams` to entirely skip side-effects. This can be used to slightly increase the performance of the overall import or to perform some system troubleshooting.
```
Comment (vilkg) [inline src/commonmark/en/content/developer/web-api/new-tracker.md]: I would suggest not to use `TrackerImportParams` here and instead say that it's a query param that can be used in the request, maybe even providing example.

**Review by stian-sandvold — APPROVED** [review]

File: src/commonmark/en/content/developer/web-api/new-tracker.md (hunk 3 above)
Comment (stian-sandvold) [reply src/commonmark/en/content/developer/web-api/new-tracker.md]: I will cover the param in a different section. I removed the "TrackerImportParams" and just refer to them as "parameter".

**Review by vilkg — APPROVED** [review]

**Pull request merged**